website/forms/php/filters/general/http-request-timeout.md

```diff
@@ -1,6 +1,13 @@
 ---
 id: http-request-timeout
-title: Http request timeout
+title: HTTP request timeout
 ---
 
-*Coming soon*
+This filter can be used to change the cURL timeout for the file upload, useful if you have to upload large files.
```

Comment:
It might be useful to add the time unit to the description... not sure if this is in `s` or `ms`

Reply:
tnx, added